### PR TITLE
feat: SNYK_LS_PATH var is now required [ROAD-1050]

### PR DESCRIPTION
**Pull request by j-luong**
### Description

_As an engineer interested in using a standalone LS binary, I want to be able to provide a path to binary. This PR will add a required env var `SNYK_LS_PATH` to achieve this._

### Checklist

- [x] Tests added and all succeed
- [x] Linted
- [x] CHANGELOG.md updated

### CODE REVIEW

File: src/snyk/common/configuration/configuration.ts [hunk 1]
```diff
@@ -171,6 +171,11 @@ export class Configuration implements IConfiguration {
     return `${authUrl.toString()}manage/snyk-code?from=vscode`;
   }
 
+  get snykLanguageServerPath(): string {
```
Comment (j-luong) [inline src/snyk/common/configuration/configuration.ts:174]: should this be hidden behind a feature flag param? If so, what should I use?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
 # Snyk Security - Code and Open Source Dependencies Changelog
 
+## [1.2.25]
+
+### Feat
```
Comment (michelkaporin) [inline CHANGELOG.md:5]: We usually have "Added", "Fixed", "Changed" categories in changelog if you look below.

File: src/snyk/common/configuration/configuration.ts (hunk 1 above)
Comment (michelkaporin) [reply src/snyk/common/configuration/configuration.ts:174]: This isn't referenced anywhere at the moment, thus no hiding should happen. Once @bastiandoetsch wires language server this config will be accessed only behind the feature flag, so all good here ✅

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+
+- Snyk Code: Adds setting a path for a custom LS binary
```
Comment (michelkaporin) [inline CHANGELOG.md:7]: This isn't "Snyk Code"specific configuration.

File: src/test/unit/common/configuration.test.ts [hunk 4]
```diff
@@ -111,6 +111,19 @@ suite('Configuration', () => {
     strictEqual(configuration.snykCodeBaseURL, customUrl);
   });
 
+  test('Snyk Code: Language Server path is returned when set', () => {
```
Comment (michelkaporin) [inline src/test/unit/common/configuration.test.ts:114]: It's not Snyk Code product specific, no need to prefix it 👍

File: CHANGELOG.md (hunk 3 above)
Comment (j-luong) [reply CHANGELOG.md:7]: I see, is there a preferred prefix here? `Snyk LSP: `? Looks like we prefix our tests with the specific product/feature. Or is no prefix fine here too

Comment (michelkaporin) [reply CHANGELOG.md:7]: Let's do `Snyk LS:`. Like the idea 👍

**Review by michelkaporin — APPROVED** [review]
First LS related work 🚀  Good start!

**Pull request merged**